app/scripts/controllers/network/network-controller.test.js

```diff
@@ -1100,7 +1100,7 @@ describe('NetworkController', () => {
     });
 
     describe('when the request for the latest block responds with null', () => {
-      it('stores null as whether the network supports EIP-1559', async () => {
+      it('persists false to state as whether the network supports EIP-1559', async () => {
```

Comment:
In order to satisfy a lint error, I've changed `_determineEIP1559Compatibility` to use `?.` when checking for the latest block. This ended up simplifying the type as well so that it always returns either true or false, never null.